tools/etc/frr/support_bundle_commands.conf

```diff
@@ -83,9 +83,38 @@ show version
 CMD_LIST_END
 
 # OSPF Support Bundle Command List
-# PROC_NAME:ospf
-# CMD_LIST_START
-# CMD_LIST_END
+PROC_NAME:ospf
+CMD_LIST_START
```

Comment:
Pls add - show ip ospf gr helper detail , show ip  ospf summary-address detail

Reply:
Don't you want "show ip ospf vrf all ..."?

Reply:
As this cmd will give brief about ospf per vrf, suggesting to add it.

Reply:
Sorry to be clear my question was: don't you want:`show ip ospf vrf all gr helper detail` instead of the suggested `show ip ospf gr helper detail`?